source/products/hansemerkur/dog-surgery/webapp.rst

```diff
@@ -0,0 +1,22 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        key: "KEY",
+        touchpoint: "tp_e2775e2996362bdaab169b8a74d12",
+        mode: "fullscreen",
+        language: "de",
+        config: {
+          adnr_number: {adnr_number},
```

Comment:
Is this a string or a number?

Reply:
Good catch!

Reply:
Should also have `ap_campaign` here

Reply:
After yesterdays change request when we removed `offer` button, then it's not needed. Because this parameter was needed for `offer` GTM event only.
Or I am missing something?

Reply:
In my book `offer` = `unpaid policy`. Can you check with Annanela for clarification?

Reply:
Yes, `ap_campaign` parameter was needed for `offer` trigger when we needed to send GTM event. 
As we don't have anymore `offer` button, this GTM event isn't triggered. There are only one button in `summary` screen - `buy` button and also `lead-link` to external HanseMerkur form.